package.json

```
@@ -7,8 +7,9 @@
     "build": "next build",
     "start": "next start",
     "test": "jest",
-    "lint": "run-s lint:js",
-    "lint:js": "next lint"
+    "lint": "run-s lint:scss",
```

The `lint:js` step got wiped out again.

```suggestion
    "lint": "run-s lint:js lint:scss",
```